deploy/docker/Dockerfile

```diff
@@ -24,26 +31,25 @@ RUN gem install fluent-plugin-s3
        # && gem install fluent-plugin-azure-storage-append-blob
 
 # FluentD plugins from RubyGems
-RUN gem install fluent-plugin-systemd -v 1.0.2 \
-       && gem install fluent-plugin-record-modifier -v 2.0.1 \
-       && gem install fluent-plugin-kubernetes_metadata_filter -v 2.4.1 \
-       && gem install fluent-plugin-sumologic_output -v 1.7.1 \
-       && gem install fluent-plugin-concat -v 2.4.0 \
-       && gem install fluent-plugin-rewrite-tag-filter -v 2.2.0 \
-       && gem install fluent-plugin-prometheus -v 1.6.1
+RUN gem install \
+       fluent-plugin-systemd:1.0.2 \
+       fluent-plugin-record-modifier:2.0.1 \
+       fluent-plugin-kubernetes_metadata_filter:2.4.1 \
+       fluent-plugin-sumologic_output:1.7.1 \
+       fluent-plugin-concat:2.4.0 \
+       fluent-plugin-rewrite-tag-filter:2.2.0 \
+       fluent-plugin-prometheus:1.6.1
 
-# FluentD plugins from this repository
-RUN gem install --local fluent-plugin-prometheus-format \
-       && gem install --local fluent-plugin-kubernetes-sumologic \
-       && gem install --local fluent-plugin-enhance-k8s-metadata \
-       && gem install --local fluent-plugin-datapoint \
-       && gem install --local fluent-plugin-protobuf \
-       && gem install --local fluent-plugin-events
+COPY gems/fluent-plugin*.gem ./
 
-# Run dependencies
-RUN apt-get update \
- && apt-get install -y curl \
- && apt-get install -y unzip
+# FluentD plugins from this repository
+RUN gem install \
+       --local fluent-plugin-prometheus-format \
+       --local fluent-plugin-kubernetes-sumologic \
+       --local fluent-plugin-enhance-k8s-metadata \
+       --local fluent-plugin-datapoint \
+       --local fluent-plugin-protobuf \
+       --local fluent-plugin-events
 
 # Install Terraform
 ENV TERRAFORM_VERSION 0.12.26
```

Comment:
Can we install Terraform before local plugins too?

Reply:
done